README.md

```diff
@@ -403,6 +403,35 @@ db.on('put', function (key, value) {
 See the [`CONTRIBUTING.md`](https://github.com/Level/community/blob/master/CONTRIBUTING.md) file for more details.
 
 <a name="license"></a>
+## Contributors
```

Comment:
This is a bad spot since we have `<a name="license"></a>` in the wrong place. We should probably remove these anyway and just use markdown links. WDYT @vweevers ?

Reply:
Made a PR to `levelup` to show what I mean https://github.com/Level/levelup/pull/534

Reply:
I feel like we can do without the Contributors section.